homeassistant/strings.json

```diff
@@ -51,6 +51,7 @@
         "already_configured_account": "Account is already configured",
         "already_configured_service": "Service is already configured",
         "already_configured_device": "Device is already configured",
+        "already_in_progress": "Config flow is already in progress",
```

Comment:
Is "Config" a real word ? Should we use "Configuration" ?

Reply:
Well config is the abbreviation of configuration so it is a word, however I am not sure if an end user has to know what config flow means. Maybe I should change it to something like that:
```suggestion
        "already_in_progress": "Configuration process is already in progress",
```
@MartinHjelmare what do you think?

Reply:
We should use the word flow as that's the official term we use in Home Assistant.

Reply:
Sure thing, so just "Flow is already in progress"?

Reply:
Configuration Flow is the full word.

Reply:
```suggestion
        "already_in_progress": "Configuration flow is already in progress",
```